SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1506 @@
+		$ppmEnvironmentalDetails.Add($result)
+		
+		# Retrieve NSX Manager password policy details
+		$nsxtManagerDetails = Get-NsxtServerDetail -fqdn $ppmVariables.sddcManagerFqdn -username $ppmVariables.sddcManagerUser -password $ppmVariables.sddcManagerPass -domain $ppmVariables.sddcDomainName -listNodes
+		foreach ($nsxtManagerNode in $nsxtManagerDetails.nodes) {
+			$result = getNsxtManagerPasswordPolicy  -server $nsxtManagerNode.fqdn -user $nsxtManagerDetails.adminUser -pass $nsxtManagerDetails.AdminPass
+			$result.productAttributes | Add-Member -notepropertyname "driftAlarm" -notepropertyvalue ""
+			$result.productAttributes | Add-Member -notepropertyname "driftMessage" -notepropertyvalue ""
+			$ppmEnvironmentalDetails.Add($result)
+		}	
+
+		# Retrieve NSX Edge password policy details
+		$results = getNsxtEdgeNodePasswordPolicy -server $nsxtManagerDetails.fqdn -user $nsxtManagerDetails.adminUser -pass $nsxtManagerDetails.AdminPass
+		foreach ($result in $results) {
+			$result.productAttributes | Add-Member -notepropertyname "driftAlarm" -notepropertyvalue ""
+			$result.productAttributes | Add-Member -notepropertyname "driftMessage" -notepropertyvalue ""
+			$ppmEnvironmentalDetails.Add($result)
+		}
+
+		# Retrieve WSA password policy details
+		if ($ppmVariables.wsaFqdn -and $ppmVariables.wsaAdminUser -and $ppmVariables.wsaAdminPass) {
+			Write-LogMessage -type INFO -Message "Retrieving Workspace ONE Access appliance password policy..."
+			$result = getWSAPasswordPolicyAll -server $ppmVariables.wsaFqdn -user $ppmVariables.wsaAdminUser -pass $ppmVariables.wsaAdminPass
+			$result.productAttributes | Add-Member -notepropertyname "driftAlarm" -notepropertyvalue ""
+			$result.productAttributes | Add-Member -notepropertyname "driftMessage" -notepropertyvalue ""
+			$ppmEnvironmentalDetails.Add($result)
+			Write-LogMessage -type INFO -Message "Retrieving Workspace ONE Access appliance password policy completed."
+		} else {
+			Write-LogMessage -Type WARNING -Message "Workspace ONE Access appliance skipped. Login details are not provided." -Colour Yellow
+		}
+
+		$ppmVariables.isEnvDetailSet = $true
+		Write-LogMessage -type INFO -Message "Retrieving environment details completed." -Colour Green
+	}
+}
+
+################################################################################
+####   Print current environment password policies settings as HTML file	####
+################################################################################
+
+Function exportReportHTML {
+
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$ppmVariables,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmStandardConfigValues,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmEnvironmentalDetails,
+		[Parameter (mandatory = $false)] [Switch]$docCompare = $false
```

Comment:
There are 2 instances of `doComparison` vs `docCompare`.

Reply:
addressed in 7c76aadcc899391eebaf87062fd4f1e38b005834